frontend/javascripts/dashboard/folders/details_sidebar.tsx

```diff
@@ -172,15 +177,15 @@ function DatasetDetails({ selectedDataset }: { selectedDataset: APIDatasetCompac
             </Tag>
           )}
         </div>
+        {fullDataset && (
```

Comment:
Open question? Is it ok to move the tags into the "spinner area"? Previously they weren'T but, the metadata should be within the "spinner area" as it depends on `fullDataset` and IMO the newly added UI element should appear after the already known tags.

Reply:
sure, why not :+1: